File: CS/12_Tables/RemoveRowsAndColumns.cs
using Spire.Presentation;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace RemoveRowsAndColumns
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\RemoveRowsAndColumns.pptx");

            //Get the table in PPT document
            ITable table = null;
            foreach (IShape shape in presentation.Slides[0].Shapes)
            {
                if (shape is ITable)
                {
                    table = (ITable)shape;

                    //Remove the second column
                    table.ColumnsList.RemoveAt(1, false);

                    //Remove the second row
                    table.TableRows.RemoveAt(1, false);
                }
            }
            //Save and launch the document
            presentation.SaveToFile("RemoveRowsAndColumns_result.pptx", FileFormat.Pptx2010);
            System.Diagnostics.Process.Start("RemoveRowsAndColumns_result.pptx");
        }
    }
}

File: VB/12_Tables/RemoveRowsAndColumns.vb
Imports Spire.Presentation
Imports System.ComponentModel
Imports System.Text

Namespace RemoveRowsAndColumns
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()
			presentation.LoadFromFile("..\..\..\..\..\..\Data\RemoveRowsAndColumns.pptx")

			'Get the table in PPT document
			Dim table As ITable = Nothing
			For Each shape As IShape In presentation.Slides(0).Shapes
				If TypeOf shape Is ITable Then
					table = CType(shape, ITable)

					'Remove the second column
					table.ColumnsList.RemoveAt(1, False)

					'Remove the second row
					table.TableRows.RemoveAt(1, False)
				End If
			Next shape
			'Save and launch the document
			presentation.SaveToFile("RemoveRowsAndColumns_result.pptx", FileFormat.Pptx2010)
			Process.Start("RemoveRowsAndColumns_result.pptx")
		End Sub
	End Class
End Namespace
